prow/getting_started_deploy.md

```diff
@@ -202,49 +215,20 @@ to start receiving GitHub events!
 
 ## Add the webhook to GitHub
 
-### Add your first webhook
-
-You have two options to do this:
-
-1. You can do this with the `update-hook` utility:
-
-```sh
-# Note /path/to/hook/secret and /path/to/github/token from earlier secrets step
-# Note the an.ip.addr.ess from previous ingress step
-
-# Ideally use https://bazel.build, alternatively try:
-#   go get -u k8s.io/test-infra/experiment/update-hook && update-hook
-$ bazel run //experiment/update-hook -- \
-  --hmac-path=/path/to/hook/secret \
-  --github-token-path=/path/to/github/token \
-  --hook-url http://an.ip.addr.ess/hook \
-  --repo my-org/my-repo \
-  --repo my-whole-org \
-  --confirm=false  # Remove =false to actually add hook
-```
-
-Look for the `http://an.ip.addr.ess/hook` you added above.
-A green check mark (for a ping event, if you click edit and view the details of the event) suggests everything is working!
-
-2. If you do not want to use the `update-hook` utility, you can go the GitHub web page and add the hook manually:
-
-- Go to your org or repo and click `Settings -> Webhooks`, and click `Add webhook`.
-- Change the `Payload URL` to `http://an.ip.addr.ess/hook` you are planning to add.
-- Change the `Content type` to `application/json`, and change your `Secret` to the `hmac-path` secret you created above.
-- Change the trigger to `Send me **everything**`.
-- Click `Add webhook`.
-
-### Use `hmac` tool to manage webhooks and hmac tokens (recommended)
+To set up the webhook, you have to go the the GitHub UI and edit your app. Update
+the `Webhook URL` property to `https://prow.<<yourdomain.com>>/hook`. Use the URL
+shown above when getting the `Ingress`.
 
-If you need to configure webhooks for multiple orgs or repos, the manual process does not work that well as it
-can be error-prone, and it'll be painful when you want to replace the hmac token if it is accidentally leaked.
+## Install app
```

Comment:
Suggest change this to something like "Configure webhook for a repo"

Reply:
Updated, WDYT?

Reply:
lgtm